spec/suites/map/MapSpec.js

```diff
@@ -2073,6 +2073,21 @@ describe("Map", function () {
 			expect(map.getCenter().distanceTo(center)).to.be.lessThan(5);
 		});
 	});
+	describe("#panBy", function () {
+
+		it("throws if map is not set before", function () {
+			expect(function () {
+				map.panBy();
+			}).to.throwError();
+		});
+
+		it("pans the map by the given number of pixels", function () {
+			var center = L.latLng(10, 10);
+			map.setView(center);
+			var change = L.point(200, 300);
+			expect(map.panBy(change)).to.be(map);
```

Comment:
The test should be check if the map has successfull and correct panned, but in the `expect` you only test if the return value of `map.panBy()` == `map`